dataplane/standalone/saiserver/switch_test.go

```diff
@@ -102,15 +102,15 @@ func TestCreateSwitch(t *testing.T) {
 	}
 }
 
-func newTestSwitch(t testing.TB) (saipb.SwitchClient, *attrmgr.AttrMgr, func()) {
+func newTestServer(t testing.TB, newSrvFn func(mgr *attrmgr.AttrMgr, srv *grpc.Server)) (grpc.ClientConnInterface, *attrmgr.AttrMgr, func()) {
```

Comment:
could newSrvFn be an option in case a test wants to use a default fn like before?

Reply:
made the func optional by passing nil